README.md

```diff
@@ -59,7 +59,7 @@ Alternatively, if you just want to just silence proptype errors when using [esli
 ### Usage as a Decorator
 `react-tracking` is best used as a `@decorator()` using the [babel decorators plugin](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy).
 
-The decorator can be used on React Classes and on methods within those classes.
+The decorator can be used on React Classes and on methods within those classes. If you use it on methods within these classes, make sure to decorate the class as well. 
 
```

Comment:
I added this here because, initially, it seems optional. You can either/or decorate methods and classes. 

For a while we were decorating methods, yet running into runtime errors (because naturally, the tracking props hadn't been added to the component). Thought it might help to explicitly mention, if you decorate methods, also to decorate the class itself so those tracking props are exposed :).

Reply:
Yes, good call. Thank you!